### PR TITLE
R4 Allergy Intolerance: Added revinclude as search parameters

### PR DESCRIPTION
**Pull request by ramya1096**
…rance

Description
Added search parameter "_revinclude" and example of response body.

Screenshot 2021-04-29 at 1 40 06 PM

Screenshot 2021-04-29 at 1 40 48 PM

PR Checklist
----
- [X] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/general-clinical/allergy-intolerance.md [hunk 1]
```diff
@@ -76,6 +86,21 @@ Search for AllergyIntolerances that meet supplied query parameters:
...
 
+### Example with RevInclude
+
+### Authorization Types
+
+<%= authorization_types(provider: true, system: true) %>
+
+#### Request
+
+    GET https://fhir-ehr.devcerner.com/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/AllergyIntolerance?_id=74771957,S-75479691&_revinclude=Provenance:target
```
Comment (ashu-bdvt) [inline content/millennium/r4/general-clinical/allergy-intolerance.md:97]: Update the tenant to "https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d" and in other places as well as relevant.

File: lib/resources/example_json/r4_examples_allergy_intolerance.rb [hunk 2]
```diff
@@ -84,6 +84,329 @@ module Resources
...
     }.freeze
 
+    R4_ALLERGY_INTOLERANCE_REVINCLUDE_BUNDLE ||= {
+      'resourceType': 'Bundle',
+      'id': '65ee4c79-3514-4eab-b606-10ad56fb0d5d',
+      'type': 'searchset',
+      'link': [
+        {
+          'relation': 'self',
+          'url': 'https://fhir-ehr.devcerner.com/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/AllergyIntolerance?_id=74771957%2CS-75479691&_revinclude=Provenance%3Atarget'
```
Comment (ashu-bdvt) [inline lib/resources/example_json/r4_examples_allergy_intolerance.rb:94]: Update tenant to https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d

File: lib/resources/example_json/r4_examples_allergy_intolerance.rb [hunk 3]
```diff
@@ -84,6 +84,329 @@ module Resources
...
+        }
+      ],
+      'entry': [
+        {
+          'fullUrl': 'https://fhir-ehr.devcerner.com/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/AllergyIntolerance/74771957',
```
Comment (ashu-bdvt) [inline lib/resources/example_json/r4_examples_allergy_intolerance.rb:99]: Update tenant to https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d

**Review by ashu-bdvt — CHANGES_REQUESTED** [review]

File: content/millennium/r4/general-clinical/allergy-intolerance.md [hunk 4]
```diff
@@ -59,6 +59,16 @@ Search for AllergyIntolerances that meet supplied query parameters:
  `_id`              | This or `patient`, if populated all other parameters are not allowed | [`token`]     | The logical resource id associated with the resource. Example: `12345`
  `patient`          | This or `_id`                                                        | [`reference`] | Who the sensitivity is for. Example: `12345`
  `clinical-status`  | No                                                                   | [`token`]     | The clinical status of the allergy or intolerance. Example: `active`, `inactive`, `resolved`
+ `_revinclude`      | No                                                                   | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example:_revinclude=Provenance:target
+
+Notes:
+
+The _revinclude parameter may be provided once with the value Provenance:target. Example: _revinclude=Provenance:target
+
+The _revinclude parameter may be provided with the _id parameter. Example: _id=74771957,S-75479691&_revinclude=Provenance:target
```
[the remark this answers is not in the document]
Comment (ramya1096) [reply content/millennium/r4/general-clinical/allergy-intolerance.md:68]: changes addressed

File: content/millennium/r4/general-clinical/allergy-intolerance.md (hunk 1 above)
Comment (ramya1096) [reply content/millennium/r4/general-clinical/allergy-intolerance.md:97]: changes addressed

File: lib/resources/example_json/r4_examples_allergy_intolerance.rb (hunk 2 above)
Comment (ramya1096) [reply lib/resources/example_json/r4_examples_allergy_intolerance.rb:94]: changes addressed

File: lib/resources/example_json/r4_examples_allergy_intolerance.rb (hunk 3 above)
Comment (ramya1096) [reply lib/resources/example_json/r4_examples_allergy_intolerance.rb:99]: changes addressed

**Review by ashu-bdvt — CHANGES_REQUESTED** [review]
Please update the screenshots to show the latest changes.

**Review by ashu-bdvt — APPROVED** [review]

**Review by UjwalaBheema — APPROVED** [review]

**Comment by Coffeevee** [comment]
**Live Validation**

Screen Shot 2021-05-07 at 10 22 03 AM

**Pull request merged**